2017/DVE-2017-0011.md

```diff
@@ -0,0 +1,49 @@
+# DVE-2017-0011: GitBook CDN nameservers return A records on any QTYPE query
+
+## Description
+
+A CDN service running 'gitbook.com' zone returns A record on every query.
+
+## Evidence
+
+### AAAA query
+
+```
+$ kdig +edns=0 IN AAAA cdn.gitbook.com. @ns2.gitbook.me.
+;; ->>HEADER<<- opcode: QUERY; status: NOERROR; id: 40264
+;; Flags: qr rd; QUERY: 1; ANSWER: 1; AUTHORITY: 0; ADDITIONAL: 0
+
+;; QUESTION SECTION:
+;; cdn.gitbook.com.    		IN	AAAA
+
+;; ANSWER SECTION:
+cdn.gitbook.com.    	3600	IN	A	95.85.1.232
+
+;; Received 64 B
+;; Time 2017-02-17 11:53:14 CET
+;; From 107.170.33.140@53(UDP) in 109.4 ms
+```
+
+### Non-authoritative zone query
+
+```
```

Comment:
Fixed. And I have also noticed they don't set the 'AA' bit. Anyway I got reponse from their side pretty quick, so this will hopefully get fixed soon.